### PR TITLE
Add / remove card in existing deck

### PR DESCRIPTION
**Pull request by danzo3000**
restyles the createCard, adds the ability to add a new card to an existing deck, and also to remove a card from a deck

### CODE REVIEW

File: utils/api.js [hunk 1]
```diff
@@ -40,3 +40,9 @@ export const getDeckByID = (id) => {
     return data.cards;
   });
 };
+
+export const deleteCard = (id) => {
```
Comment (RogerioSoares96) [inline utils/api.js:44]: @Nanobot1514 By deleting ont he BE, will it also be removed from the decks cards array ?

**Comment by RogerioSoares96** [comment]
Other than the comment that I left for @Nanobot1514 It looks good to merge :)

**Comment by RogerioSoares96** [comment]
great work !

File: utils/api.js (hunk 1 above)
Comment (Nanobot1514) [reply utils/api.js:44]: @RogerioSoares96 yes, the function in the BE uses $pull which removes the id reference from the decks

File: components/SingleDeck.jsx [hunk 2]
```diff
@@ -24,16 +27,37 @@ const SingleDeck = ({ route }) => {
...
     setIsModalVisible(true);
     setCardID(_id);
   };
 
+  const handleDeletePress = (_id) => {
+    deleteCard(_id).then(() => {
+      console.log("card deleted SingleDeck");
+      setDeck((currentDeck) => {
+        return currentDeck.filter((card) => {
+          return card._id !== _id;
```
Comment (RogerioSoares96) [inline components/SingleDeck.jsx]: Well done optismistically rendered !

**Pull request merged**